packages/terra-infinite-list/src/InfiniteList.jsx

```diff
@@ -140,9 +149,18 @@ class InfiniteList extends React.Component {
     if (!this.props.isFinishedLoading && !this.hasRequestedItems && !this.isRenderingNew && this.props.onRequestItems) {
       this.hasRequestedItems = true;
       this.props.onRequestItems();
+      this.updateAriaLiveLoadingStatus();
     }
   }
 
+  updateAriaLiveLoadingStatus() {
+    this.ariaLiveStatus = this.props.intl.formatMessage({ id: 'Terra.InfiniteList.loading' });
```

Comment:
Should this loading message be a prop set by users? ie. 'Loading allergies' etc.

Reply:
We could make it optional so even if unset the component says loading.

Reply:
Yeah we can make it prop based and default back to the translated string we provide.

Reply:
Updated: https://github.com/cerner/terra-framework/pull/732/commits/e2c43f48898947aead1be8f4526988159e07fbf1